contrib/babelfishpg_tsql/sql/sys_views.sql

```diff
@@ -3004,3 +3004,39 @@ SELECT
     CAST(0 as sys.BIT) AS is_ambiguous
 WHERE FALSE;
 GRANT SELECT ON sys.sql_expression_dependencies TO PUBLIC;
+
+
+create or replace view sys.sequences as
+select
+    CAST(p.relname as sys.sysname) as name
+  , CAST(p.oid as int) as object_id
+  , CAST(null as int) as principal_id
+  , CAST(s.schema_id as int) as schema_id
+  , CAST(0 as int) as parent_object_id
+  , CAST('SO' as char(2)) as type
+  , CAST('SEQUENCE_OBJECT' as sys.nvarchar(60)) as type_desc
+  , CAST(null as sys.datetime) as create_date
+  , CAST(null as sys.datetime) as modify_date
+  , CAST(0 as sys.bit) as is_ms_shipped
+  , CAST(0 as sys.bit) as is_published
+  , CAST(0 as sys.bit) as is_schema_published
+  , CAST(ps.seqstart as sql_variant ) as start_value
+  , CAST(ps.seqincrement as sql_variant ) as increment
+  , CAST(ps.seqmin as sql_variant  ) as minimum_value
+  , CAST(ps.seqmax as sql_variant ) as maximum_value
+  , CASE ps.seqcycle when 't' then CAST(1 as sys.bit) else CAST(0 as sys.bit) end as is_cycling
+  , CAST(ps.seqcache as sys.bit ) as is_cached
+  , CAST(NULL as int ) as cache_size
+  , CAST(ps.seqtypid as int ) as system_type_id
+  , CAST(ps.seqtypid as int ) as user_type_id
+  , CAST(0 as sys.tinyint ) as precision
+  , CAST(0 as sys.tinyint ) as scale
+  , CAST('ABC' as sql_variant  ) as current_value
+  , CAST(0 as sys.bit ) as is_exhausted
+  , CAST('ABC' as sql_variant ) as last_used_value
```

Comment:
it can be `ps.last_value`

Reply:
pg_sequence does not have column last_value

Reply:
Official doc for PG suggests so. https://www.postgresql.org/docs/15/view-pg-sequences.html

Reply:
Oh! It is _pg_sequences_, not _pg_sequence_. Can we use _pg_sequences_ instead?

Reply:
I think we cannot use directly as it does not a primary key to do the join.